### ISSUE TITLE
Feature request: flexible number presentation (1,000,000.123, not 1000000.123)

### ISSUE THREAD

**Issue by mykaul** [tobami/codespeed#137]
With some values of benchmarks (such as IOPS, packets-per-second, etc.), it is easier to read if you use a notation such as 1,234,567.89 and not 1234567.89

**Comment by Ismael-VC** [comment]
This can be done with the humanize tag:

* https://docs.djangoproject.com/en/1.11/ref/contrib/humanize

**Comment by str4d** [comment]
That would work for the Changes page, but not the Timeline or Comparison pages (the values there are rendered by jqPlot, not Django).

**Comment by tobami** [comment]
@str4d yeah, I agree about the humanize tag. It's a valid point though, much easier to read if formatted properly
